http_logs/challenges/common/default-schedule.json

```diff
@@ -0,0 +1,183 @@
+{
```

Comment:
With this level of indentation the rendered track by Rally looks like:

```
  "challenges": [
        {
      "name": "append-no-conflicts",
      "description": "Indexes the whole document corpus using Elasticsearch default settings. We only adjust the number of replicas as we benchmark a single node cluster and Rally will only start the benchmark if the cluster turns green. Document ids are unique so all index operations are append only. After that a couple of queries are run.",
      "default": true,
      "schedule": [
        {
  "operation": "delete-index"
},
{
  "operation": {
    "operation-type": "create-index",
    "settings": {}
  }
},
```

to make things look proper when rendered internally by Rally (you can check those rendered files in your $TMPDIR, they are useful for debugging) we should indent by 8 characters